CHANGELOG.md

```diff
@@ -4,11 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [master] - 2019-10
+## [3.4.0] - 2019-11-15
 ### Added
+- M1 to M2 Stored Card migration tool
+  - New `bin/magento` console command to connect to your remote M1 database and potentially copy across customers
```

Comment:
Can we list the command?

Reply:
New command is outlined in the README :) But yes, I can put it here as well!